contracts/SlashIndicator.sol

```diff
@@ -197,6 +206,19 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function downtimeSlash(address validator, uint256 count) external override onlyValidatorContract {
+    _downtimeSlash(validator, count);
+  }
+
+  function _downtimeSlash(address validator, uint256 count) internal {
+    try IStakeHub(STAKE_HUB_ADDR).downtimeSlash(validator) {
+    } catch Error(string memory reason) {
+      emit failedFelony(validator, count, bytes(reason));
+    } catch (bytes memory lowLevelData) {
+      emit failedFelony(validator, count, lowLevelData);
+    }
+  }
+
   function submitFinalityViolationEvidence(FinalityEvidence memory _evidence) public onlyInit onlyRelayer {
```

Comment:
In the future, we dont have relayer anymore, so no `onlyRelayer` anymore?

Reply:
Removed

Reply:
The `BSC_RELAYER_REWARD`  const is no longer used.

Reply:
Removed